lib/tapioca/helpers/active_record_column_type_helper.rb

```diff
@@ -63,9 +63,19 @@ def do_not_generate_strong_types?(constant)
         !(constant.singleton_class < Object.const_get(:StrongTypeGeneration))
   end
 
+  sig { params(type: String).returns(String) }
+  def as_nilable_type(type)
```

Comment:
Agreed, but I would like to leave deduplication to a later effort, since I couldn't readily find a good home for this method. It feels like we need a module to help with constructing union/intersection/nilable, etc types as a more general refactor.